pkg/ruler/api.go

```diff
@@ -119,6 +125,26 @@ func respondError(logger log.Logger, w http.ResponseWriter, msg string) {
 	}
 }
 
+func respondClientError(logger log.Logger, w http.ResponseWriter, msg string) {
```

Comment:
what's the different of this function with respondError?

Reply:
That function returns 400 vs respondError returns 500. I added the function to be consistent with [respondAccepted()](https://github.com/cortexproject/cortex/blob/master/pkg/ruler/api.go#L308)

Reply:
Changed this to return 400 and changed to return bad request/data.